submissions/Oleksandr-Heleta/OOP_exercise/index.js

```diff
@@ -14,23 +14,21 @@ class Inhabitants {
       this.gender = gender;
       this.legs = legs;
       this.say = say;
-      this.propertiesOrder = ['species', 'name', 'gender', 'legs', 'say']
-   }
-
-   addProperty(property, order) {
-      this.propertiesOrder.splice(order, 0, property);
    }
 
    createString() {
-      return this.propertiesOrder.map((property) => this[property] || 'none');
+      return `${this.species}, ${this.name}, ${this.gender}, ${this.legs}, ${this.say};`;
    }
```

Comment:
Changed render method

Reply:
Fine. Now, make this frgament DRYer using `Array#join` method.

Reply:
Now the method name promises to create a string, but it creates something else.
Yet there is way to keep the promise.